frontend/src/views/AuthPage.vue

```diff
@@ -0,0 +1,184 @@
+<script>
+import authService from '../api/auth';
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { useUserStore } from '@/stores/user';
+
+export default {
+  name: 'auth-page',
+  data: () => {
+    return { isToggled: false };
+  },
```

Comment:
Nitpick: since the `setup` method is already being used, `isToggled` can be moved to it and be created with the `ref` method.